garrysmod/gamemodes/sandbox/gamemode/spawnmenu/creationmenu/content/contentheader.lua

```diff
@@ -27,7 +27,7 @@ end
 
 function PANEL:SizeToContents()
 
-	local w, h = self:GetContentSize()
+	local w, _ = self:GetContentSize()

```

Comment:
And here.

Reply:
I think they should remain untouched. Changing it lessens readability.

Reply:
I disagree - if they aren't used, they shouldn't be defined.